index.html

```diff
@@ -603,3 +612,5 @@
 [fire a simple event]: https://html.spec.whatwg.org/multipage/webappapis.html#fire-a-simple-event
```

Comment:
I prefer not to touch that in current PR, and change it later. What's the current definition for "fire an event"?

(Also: we need a build process for all specs that will warn of definitions that go nowhere...)